layouts/partials/helpers/image.html

```diff
@@ -3,10 +3,12 @@
 {{- .root.page_scratch.Set "image" (printf "images/%s" $image) -}}
 
 {{/* Page specific resource */}}
-{{- $location := (printf "%s/%s" .root.fallthrough.page_path $image) -}}
-{{- if (fileExists (printf "content/%s" $location)) -}}
-  {{/* special case index: trim _index/ from url */}}
-  {{- .root.page_scratch.Set "image" $location -}}
+{{- if .root.page -}}
+  {{- $location := (printf "%s/%s" .root.page.Dir $image) -}}
+  {{- if (fileExists (printf "content/%s" $location)) -}}
+    {{/* special case index: trim _index/ from url */}}
```

Comment:
We don't actually trim the `_index/` anymore. some in the fragment specific resource code. Or am I missing something?